server/middleware/setUpWebSecurity.ts

```diff
@@ -1,35 +1,45 @@
-import express, { Router, Response } from 'express'
+import express, { Router, Request, Response, NextFunction } from 'express'
 import helmet from 'helmet'
 import crypto from 'crypto'
+import config from '../config'
 
 export default function setUpWebSecurity(): Router {
   const router = express.Router()
 
-  router.use((req, res, next) => {
+  router.use((_req: Request, res: Response, next: NextFunction) => {
     res.locals.cspNonce = crypto.randomBytes(16).toString('base64')
     next()
   })
 
-  // Secure code best practice - see:
-  // 1. https://expressjs.com/en/advanced/best-practice-security.html,
-  // 2. https://www.npmjs.com/package/helmet
+  // Hash allows inline script pulled in from https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/template.njk
```

Comment:
followed instructions in https://github.com/ministryofjustice/hmpps-micro-frontend-components/blob/main/readme/incorporating.md